inmoose/pycombat/covariates.py

```diff
@@ -13,16 +13,19 @@
 
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 
 import logging
 import numpy as np
 import pandas as pd
-from patsy import dmatrix
+from patsy import dmatrix, DesignMatrix
 
 from ..utils import asfactor
 
-def make_design_matrix(counts, batch, group, covar_mod, full_mod, ref_batch):
```

Comment:
`full_mod` argument is lost, what is the impact??

Reply:
In my understanding it has no impact, it was a parameter to say if we wanted to take covariates into account or not. In the way I modified the function I consider that if the `covar_mod` parameter is not None it means that we want to take covariates into account